programmers/난이도별/level01.핸드폰_번호_가리기/s2uyeoii.cpp

```diff
@@ -0,0 +1,13 @@
+#include <string>
+#include <vector>
+
+using namespace std;
+
+string solution(string phone_number) {
+    
+    for(int i=0; i<phone_number.length()-4; i++)
```

Comment:
```suggestion
    for(int i = 0, len = phone_number.length();  i < len - 4; i++)
```
length를 계산하는 메소드가 조건에 있을 경우 매 반복문 마다 length()를 실행하게 됩니다! 그래서 length같은 것들은 변수로 할당해서 재사용할 수 있도록 만들어주세요!